tests/conftest.py

```diff
@@ -41,6 +54,19 @@ def notebook_service(docker_ip, docker_services):
     return url, token
 
 
+@pytest.fixture(scope="session", autouse=True)
+def install_package(docker_compose):
+    # assurance for host user UID other that 1000
+    command = build_command(
```

Comment:
I don't understand why are you not using the `aiidalab_exec` fixture here?

Reply:
I tried but the "function" scope fixture is not allowed to be used for the "session" scope fixture.

Reply:
But since we are execute the command for one container that we launched from start, I think maybe the best practice is to also set the `aiidalab_exec` as a session scope fixture? Then every thing is simple. I'll give it a try.